environment.js

```diff
@@ -0,0 +1,26 @@
+var path = require('path');
+var fs = require('fs');
+var mkdirp = require('mkdirp');
+var glob = require('glob');
+
+export.ensureDirectory = function(directoryName) {
+  if (!fs.existsSync(directoryName)) {
+    mkdirp.sync(directoryName);
+  }
+};
+
+export.loadSpecifications = function(mocha, directory) {
+   function addSpecs(error, files) {
```

Comment:
Okay, I understand. Maybe, in this case, you could take advantage of hoisting and place the glob function first as for me, this would be 100% more readable.